assets/js/components/HealthSummary/HomeHealthSummary.jsx

```diff
@@ -52,7 +52,10 @@ const healthSummaryTableConfig = {
       render: (content, item) => {
         const linkToCluster = `/clusters/${item.clusterId}`;
         return (
-          <Link to={linkToCluster}>
+          <Link
+            to={linkToCluster}
+            style={{ pointerEvents: item?.clusterId ? 'auto' : 'none' }}
+          >
             <HealthIcon health={content} centered={true} />
           </Link>
```

Comment:
You can use a ternary and render the link conditionally

```suggestion
          item.clusterId ? (
            <Link
              to={linkToCluster}
              style={{ pointerEvents: item?.clusterId ? 'auto' : 'none' }}
            >
              <HealthIcon health={content} centered={true} />
            </Link>
          ) : (
            <HealthIcon health={content} centered={true} />
          )
```